ansible/README.md

```diff
@@ -14,7 +14,7 @@ can be real hardware, VMs, things in a public cloud, etc. Anything that you can
 
 ### Configure inventory
 
-Add the system information gathered above into the 'inventory' file, or create a new inventory file for the cluster.
+Add the system information gathered above into the 'inventory' file, or create a new inventory file for the cluster. Place the 'inventory' file into the `./inventories` directory.
```

Comment:
I have cherry picked your commit. Once #967 gets merged I will rebase this PR and remove your commit from it.